### PR TITLE
Adding Vivian as staff member

### PR DESCRIPTION
**Pull request by vivian271**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
\ No newline at end of file
+gem 'github-pages', group: :jekyll_plugins
+gem "webrick", "~> 1.7"
```
Comment (gallettilance) [inline Gemfile:3]: why is this module needed?

Comment (vivian271) [reply Gemfile:3]: just for running my code locally. i can remove it if you want

Comment (gallettilance) [reply Gemfile:3]: yes please

Comment (vivian271) [reply Gemfile]: done!

File: Gemfile [hunk 2]
```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
\ No newline at end of file
+gem 'github-pages', group: :jekyll_plugins
```
Comment (gallettilance) [inline Gemfile:2]: Nit: please keep the newline at the end of the file

Comment (vivian271) [reply Gemfile]: done! sorry for the extra work!

**Review by gallettilance — APPROVED** [review]

**Pull request merged**